### PR TITLE
LPS-150627 [CP2.0] [Impedibug] Fix Developer Keys 6.1 Download

### PR DESCRIPTION
**Pull request by thaynam**

### CODE REVIEW

**Review by claricebrasil — APPROVED** [review]
Great job!

**Review by joanagrego — APPROVED** [review]

**Comment by jairmedeiros** [comment]
ci:forward

**Review by Lucas2691 — APPROVED** [review]

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/layouts/DeveloperKeysLayout/Inputs/index.js [hunk 1]
```diff
@@ -77,11 +77,12 @@ const DeveloperKeysInputs = ({
 	}, [dxpVersion, listType]);
 
 	const developerKeyDownload = async () => {
+		const selectedVersionSplitted = selectedVersion.split(' ')[0];
```
Comment (igordmf) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/layouts/DeveloperKeysLayout/Inputs/index.js:80]:
```suggestion
		const [selectedVersionSplitted] = selectedVersion.split(' ');
```

**Review by igordmf — COMMENTED** [review]
Nice

**Review by igordmf — APPROVED** [review]

**Comment by jairmedeiros** [comment]
ci:forward

**Pull request closed without merging**